packages/realm/bindgen/src/templates/wrapper.ts

```diff
@@ -0,0 +1,405 @@
+import { TemplateContext } from "@realm/bindgen/context";
```

Comment:
This is missing because the eslint config is different for these files: https://github.com/realm/realm-js/blob/nh/wasm/emscripten_target/packages/realm/bindgen/.eslintrc.json

We could probably delete that file and commit the `--fix` but I'll defer that to another PR.